lib/HunkView.mli

```diff
@@ -0,0 +1,23 @@
+(** This module provides functionalities for rendering different views of hunks within a patch in a diff viewer application using the Nottui library. It supports both unified and side-by-side display of changes. *)
+
+val current_hunks : Patch.t Zipper.t Lwd.var -> Nottui.ui Lwd.t
```

Comment:
Here again, the Lwd parts of this function should be removed!

Reply:
We should define how to turn Patch.t Zipper -> Nottui.ui

Should not have Lwd stuff

The Lwd part should be extracted into InteractiveViewer.ml